x-pack/plugins/siem/server/config.ts

```diff
@@ -16,6 +16,19 @@ export const configSchema = schema.object({
   maxTimelineImportExportSize: schema.number({ defaultValue: 10000 }),
   maxTimelineImportPayloadBytes: schema.number({ defaultValue: 10485760 }),
   [SIGNALS_INDEX_KEY]: schema.string({ defaultValue: DEFAULT_SIGNALS_INDEX }),
+  /**
+   * Host Endpoint Configuration
+   */
+  endpointResultListDefaultFirstPageIndex: schema.number({ defaultValue: 0 }),
+  endpointResultListDefaultPageSize: schema.number({ defaultValue: 10 }),
+
+  /**
+   * Alert Endpoint Configuration
+   */
+  alertResultListDefaultDateRange: schema.object({
+    from: schema.string({ defaultValue: 'now-15m' }),
+    to: schema.string({ defaultValue: 'now' }),
+  }),
```

Comment:
Nit: I happened to look for the docs for these settings, and discovered that there aren't any docs for these settings (or other existing `siem` plugin settings). If there's not already an open issue for it, perhaps one should be opened to ensure that docs are added for these settings.

Reply:
Thank you @jportner, I think we are going to revisit some of that and see if we still need it and if so we will document it

Reply:
@XavierM @jportner Here's an existing github issue concerning the removal of these options: https://github.com/elastic/endpoint-app-team/issues/176